templates/compose/cryptonodes.yaml.erb

```diff
@@ -5,13 +5,18 @@ services:
     image: parity/parity:v2.7.2-stable
     restart: always
     command: |
+      --mode=active
+    <%- if @config['parity']['network'] == 'testnet' -%>
```

Comment:
We shouldn't use `testnet` for Parity, there are multiple testnet networks out there.
I'd suggest simply rendering the network name here